frontend/src/views/Game.tsx

```diff
@@ -1,13 +1,44 @@
 import Editor from '@monaco-editor/react';
 import React from 'react';
+import { Problem, getProblems } from '../api/problem';
+import styles from './styles.module.css';
 
-function GamePage() {
-  return (
-    <div>
-      Game page
-      <Editor height="90vh" language="javascript" />
-    </div>
-  );
+interface GameState {
+  problems?: Problem[];
+}
+
+interface GameProps {}
```

Comment:
What's the difference between interface and type? If we use interface here should we be doing the same elsewhere, or vice versa?

Reply:
I used interfaces here because they are more ideal for defining the props & states required by React.Component. Generally types and interfaces in Typescript are becoming more similar to each other but there are specific cases where one is better than the other, i.e. types rather than interfaces should be used as arguments or return types of a function. I think it's fine to have both in our program.